packages/terra-date-time-picker/src/terra-dev-site/doc/date-time-picker/Events.2.doc.mdx

```diff
@@ -46,8 +46,8 @@ terra-date-time-picker has several callback events that are slightly different i
   * `isCompleteValue` - A boolean to indicate whether or not both the date and time values are filled out and conforms to the date/time format. If this boolean is false, the iSO attribute would be empty.
   * `isValidValue` - A boolean to indicate whether or not the complete date/time value is within the range of the minDate and maxDate props and is not an excluded date.
 
-### onClickOutside
-* The `onClickOutside` callback prop is triggered when clicking anywhere outside of the date picker to dismiss it.
+### onDismiss
+* The `onDismiss` callback prop is triggered when clicking anywhere outside of the date picker to dismiss it.
```

Comment:
`onDismiss` would get called whenever the picker closes. Even when selecting a date so it is not limited to clicking outside.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1224/commits/e5b41a65262bc94c8d9462989d7edc89c507ff74